src/wp-includes/script-loader.php

```diff
@@ -757,26 +757,26 @@ function wp_default_scripts( $scripts ) {
 	$scripts->add( 'common', "/wp-admin/js/common$suffix.js", array( 'jquery', 'hoverIntent', 'utils', 'wp-a11y' ), false, 1 );
 	$scripts->set_translations( 'common' );
 
-	$bulk_field_relations = array(
+	$bulk_action_observer_ids = array(
 		'bulk_action' => 'action',
 		'changeit'    => 'new_role',
 	);
 	did_action( 'init' ) && $scripts->localize(
 		'common',
-		'bulkFieldRelations',
+		'bulkActionObserverIds',
 		/**
 		 * Filters the array of field relations for the bulk actions.
 		 *
-		 * @since 6.7.3
+		 * @since 6.8.1
 		 *
-		 * @param array $bulk_field_relations {
-		 *      An array of field relations for the bulk actions.
+		 * @param array $bulk_action_observer_ids {
+		 *      An array of field IDs for the bulk actions.
```

Comment:
While I think that _id is accurate in the variable & filter names in terms of intent, the capitalized version here makes it seem like we mean ID as in attribute, which is only true for `new_role`. I think this should change to 'name attribute' in the filter comments. (Both here and above.)